### PR TITLE
[FrameworkBundle] Update console fixtures after #22217

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | yes (regression from #22217)
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/symfony/symfony/pull/22217#issuecomment-291497513
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

cc @fabpot 

Sorry for the inconvenience :)

### CODE REVIEW

**Comment by ro0NL** [comment]
Travis is hanging?

**Comment by fabpot** [comment]
@ro0NL No, it's just that there is a lot of activity, and the queue is quite large right now :)

**Review by chalasr — APPROVED** [review]
👍

**Comment by ro0NL** [comment]
Monitoring travis.. and PHP7 still fails. What a day...

**Comment by chalasr** [comment]
@ro0NL Not sure but I'd say `~2.8.19|~3.3`

**Comment by fabpot** [comment]
No, in 2.8, we cannot depend on anything else but 3.0. I think we dropped 3.0 in some cases though, so `~2.8.19` should be the way to go I suppose.

**Comment by ro0NL** [comment]
:tada: thx @chalasr for helping out and pointing out these issues.

File: src/Symfony/Bundle/FrameworkBundle/composer.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     },
     "require-dev": {
         "symfony/browser-kit": "~2.4|~3.0.0",
-        "symfony/console": "~2.8.8|~3.0.8",
+        "symfony/console": "~2.8.19|~3.3",
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/composer.json]: `~2.8.19|^3.2.7` ?

Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/composer.json]: ~2.8.19|~3.2.7 actually

**Comment by nicolas-grekas** [comment]
Thank you @ro0NL.

**Comment by nicolas-grekas** [comment]
merged, can you please look now at master which is still failing?

**Pull request closed without merging**